test/unit/cart.js

```diff
@@ -67,7 +70,8 @@ describe('Cart class', () => {
             title: 'test',
             variant_title: 'test2',
             line_price: 20,
-            quantity: 1
+            quantity: 1,
+            variant: { image: { src: 'cdn.shopify.com/image.jpg' } }
```

Comment:
use braces consistently. Nowhere else is are you doing `{ stuff` it's always `{stuff`. This should probably be in the linter.